pages/glossary/adjacent-areas.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Adjacent areas
+key: adjacent-areas
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+Web page areas A and B are _adjacent areas_ if at least one pixel from area A is at the distance of 1 pixel from one pixel in area B.
```

Comment:
I'm not sold on using "pixels" in these area definitions. What does it mean for an area to have pixels? Perhaps "point" is more appropriate for these definitions.